src_modules/player/index.ts

```diff
@@ -51,16 +48,7 @@ export default class PlayerModule extends Module {
             },
         );
         this.musicCommand.addSubcommands(
-            new PauseCommand(),
-            new PlayCommand(this),
-            new ResumeCommand(),
-            new SkipCommand(this),
-            new StopCommand(this),
-            new VolumeCommand(),
-            new QueueCommand(this),
-            new LyricsCommand(this),
-            new ClearCommand(this),
-            new SeekCommand(this),
```

Comment:
Don't remove the existing commands here

Reply:
Revert the removal of the existing commands here. Both the old commands and the new pop commands should be here.

Reply:
Revert the removals here